.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
Should we commit the .vscode settings config or add to gitignore?

Reply:
It can be added to .gitignore for now.